docs/sources/tempo/operations/user-configurable-overrides.md

```diff
@@ -0,0 +1,163 @@
+---
+title: User-configurable overrides
+menuTitle: Configure Tempo overrides through the user-configurable overrides API
+description: Configure Tempo overrides through the user-configurable overrides API
+weight: 90
+---
+
+# User-configurable overrides
+
+User-configurable overrides in Tempo let you change overrides for your tenant using an API.
+Instead of modifying a file or Kubernetes configmap, you (and other services relying on Tempo) can use this API to modify the overrides directly.
+
+### Architecture
+
+![user-configurable-overrides-architecture.png](./user-configurable-overrides-architecture.png)
+
+User-configurable are stored in an object store bucket managed by Tempo.
+
+{{% admonition type="note" %}}
+We recommend using a different bucket for overrides and traces storage, but they can share a bucket if needed.
+When sharing a bucket, make sure any lifecycle rules are scoped correctly to not remove data of user-configurable overrides module.
+{{% /admonition %}}
+
+Overrides of every tenant are stored at `/{tenant name}/overrides.json`:
+
+```
+overrides/
+├── 1/
+│   └── overrides.json
+└── 2/
+└── overrides.json
+```
+
+This bucket is regularly polled and a copy of the limits is kept in-memory. When requesting the overrides for a tenant, the overrides module will now:
+
+1. check this override is set in the user-configurable overrides, if so return that value
+2. check if this override is set in the runtime config (configmap), if so return that value
+3. return the default value
+
+### Supported fields
+
+User-configurable overrides are designed to be a subset of the runtime overrides. See [Overrides]{{< relref "../configuration/_index#overrides" >}} for all overrides.
+
+{{% admonition type="note" %}}
+When a field is set in both the user-configurable overrides and the runtime overrides, the value from the user-configurable overrides will be returned.
```

Comment:
might be usefult to split these notes into two and expand by what we mean by returned.

Reply:
Would you want the expanded text inside the note or as a paragraph around the note? We can make these in to separate notes easily enough.

Reply:
no strong preference, anything works as long as we highlight both of these items to reader.

Reply:
I've tweaked this a bit:
- instead of returned use "will take priority" (there might be a better way to formulate this)
- I've moved the section about priority to regular text and kept the exception in a note